multiplayer/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "arrowParens": "avoid",
+    "semi": false,

```

Comment:
nit: I am a little curious on this one, I would say most of our codebases uses semicolons / I personally find it easier to read; any particular reason to override prettier default?

Reply:
i too prefer semicolons

Reply:
I can go either way, so with-semicolons it is.  I wonder how many terabytes of semicolons exist in github's storage.